fdbserver/workloads/BulkDumping.actor.cpp

```diff
@@ -198,6 +229,36 @@ struct BulkDumping : TestWorkload {
 		return kvs;
 	}
 
+	bool keyContainedInRanges(Key key, const std::vector<KeyRange>& ranges) {
```

Comment:
Nit:
```suggestion
	bool keyContainedInRanges(const Key& key, const std::vector<KeyRange>& ranges) {
```

Reply:
Yes. I will address this in the next PR.